src/components/event-diamond.jsx

```diff
@@ -67,7 +67,7 @@ let EventDiamond = ({ talk: { title, event, speaker } }) => {
   return (
     <Diamond>
       <EventName to={eventUrl}>{event.title}</EventName>
-      <TalkTitle to={eventUrl}>{title}</TalkTitle>
+      <TalkTitle to={eventUrl} title={title}>{title}</TalkTitle>
```

Comment:
Hmm, kind of confusing. Honestly, at first i thought this is a bug. 🙂 

Maybe create a an issue to increase readability?

Reply:
I can rename this existing issue if you don't mind )) because these 2 changes about similar problem